app/controllers/spree/api/taxons_controller.rb

```diff
@@ -1,71 +0,0 @@
-module Spree
-  module Api
-    class TaxonsController < Spree::Api::BaseController
-      respond_to :json
-
-      def index
-        if taxonomy
-          @taxons = taxonomy.root.children
-        else
-          if params[:ids]
-            @taxons = Taxon.where(id: params[:ids].split(","))
-          else
-            @taxons = Taxon.ransack(params[:q]).result
-          end
-        end
-        respond_with(@taxons)
-      end
-
-      def show
```

Comment:
It looks like `spree/api/taxons#jstree` which calls `#show` was removed from OFN `Spree::Api::TaxonsController` [here](https://github.com/openfoodfoundation/openfoodnetwork/commit/c668677b8a4e2e878ee85669d9c43378c7dff128), but it is still used by the admin section for taxons. The route still works because the `spree_api` route and controller are still in place.

The next step for this set of PRs is to remove `spree_api`, and I suspect that this is not covered by ours specs. What do you think about adding back or adapting in this PR?

Reply:
I dont understand why I did this. I looked up jstree before deleting, I must have missed it...
I need to re-check the backoffice views...

Reply:
done :+1: